doc/source/dev/dev_quickstart.rst

```diff
@@ -71,7 +71,7 @@ following commands at the terminal from the base directory of your
             # Activate the environment
             source $HOME/.venvs/scipy-dev/bin/activate
             # Install python-level dependencies
-            python -m pip install numpy pytest cython pythran pybind11 meson ninja pydevtool rich-click hypothesis
+            python -m pip install numpy pytest cython pythran pybind11 meson ninja pydevtool rich-click hypothesis pooch
```

Comment:
I think this is fine, since it matches the presence of `pooch` in `environment.yml` used in the other docs tab.

Longer-term, I do wonder if we might want to switch this to one of the files in our new `requirements` folder like doing `python -m pip install -r requirements/all.txt`.

I tend to do that when I'm in a rush on a new machine and it seems ok.